jsonschema-core.md

```diff
@@ -1970,24 +1970,23 @@ The minimum number of occurrences is provided by the `minContains` keyword
 within the same schema object as `contains`. If `minContains` is absent, the
 minimum number of occurrences MUST be 1.
 
-Implementations MAY implement the dependency on `minContians` and `maxContains`
+Implementations MAY implement the dependency on `minContains` and `maxContains`
 by inspecting their values rather than reading annotations produced by those
 keywords.
```

Comment:
Do we need to say this? We've already stated elsewhere that annotations are just an explanatory device and implementations aren't required to use annotations for inter-keyword communication.

Reply:
That's to be addressed in a follow-up PR.  I'm just removing objects here.